spec/features/idv/actions/redo_document_capture_action_spec.rb

```diff
@@ -28,8 +28,10 @@
       complete_upload_step
       DocAuth::Mock::DocAuthMockClient.reset!
       attach_and_submit_images
-      complete_ssn_step
 
+      expect(current_path).to eq(idv_doc_auth_verify_step)
+      check t('forms.ssn.show')
+      expect(page).to have_content(DocAuthHelper::SSN_THAT_FAILS_RESOLUTION)
```

Comment:
The `before` block completes SSN step with failed resolution result for convenience of testing the troubleshooting options (the test case right immediately following this one), hence the `SSN_THAT_FAILS_RESOLUTION`.